src/verilog.y

```diff
@@ -4766,12 +4771,12 @@ commaVRDListE<nodep>:
         |       ',' vrdList                             { $$ = $2; }
         ;
 
-argsExprList<nodep>:            // IEEE: part of list_of_arguments (used where ,, isn't legal)
+argsExprList<nodeExprp>:            // IEEE: part of list_of_arguments (used where ,, isn't legal)
```

Comment:
```suggestion
argsExprList<nodeExprp>:        // IEEE: part of list_of_arguments (used where ,, isn't legal)
```

Similar some below. Perhaps keeping these aligned is a mistake, but not worth changing at this moment.